arpitkh96/AmazeFileManager
# Feature/style delete dialog

### Fix
Update the delete dialog style by including the following:
- Create custom view with Material-like "Directories" and "Files" text.
- Update deprecated callback methods from the MaterialDialog library [here](https://github.com/afollestad/material-dialogs/blob/master/core/src/main/java/com/afollestad/materialdialogs/MaterialDialog.java#L2194-L2246).
- Follow the Material Design guidelines for alert dialogs [here](https://material.io/guidelines/components/dialogs.html#dialogs-alerts).
- Organize code variable names and logic conditions for better readability.

### Test
1. Long-press directories and/or files.
2. Tap ***Delete*** action.
3. Notice delete dialog style.
